safe/definitions/test/test_utilities.py

```diff
@@ -444,6 +444,7 @@ def test_unique_definition_key(self):
         for duplicate_key in duplicate_keys:
             message += 'Duplicate key: %s\n' % duplicate_key
             for v in keys[duplicate_key]:
+                print v
```

Comment:
`print(v)` let's start changing our habits ;-)

Reply:
:D, forget to remove. Removed in the other PR.